mw-webapp/src/logic/reportsTable/renderCellItem/renderEditableText.tsx

```diff
@@ -0,0 +1,56 @@
+import {useState} from "react";
+import {EditableText} from "src/component/editableText/EditableText";
+import {CurrentProblemDAL} from "src/dataAccessLogic/CurrentProblemDAL";
+import {JobDoneDAL} from "src/dataAccessLogic/JobDoneDAL";
+import {MentorCommentDAL} from "src/dataAccessLogic/MentorCommentDAL";
+import {PlanForNextPeriodDAL} from "src/dataAccessLogic/PlanForNextPeriodDAL";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+
+/**
+ * Cell item props
+ */
+interface EditableTextProps {
+
+  /**
+   * Cell item's text
+   */
+  content: string;
+
+  /**
+   * Array item
+   */
+  arrayItem: JobDone | PlanForNextPeriod | CurrentProblem | MentorComment;
```

Comment:
why so business specific values? I think EditableText component should be business independent. Looks like the whole file is useless, could we gfegt rid of it and use Edutable cell directly?

*the same for other entities

Reply:
Deleted

Reply:
Deleted